### DISCUSSION TITLE
SQL sources in Examples folder not listed in breakdown

### DISCUSSION THREAD

**Discussion by gdsotirov** [github-linguist/linguist#5173] [Classification] [answered]
I have a repository with sample SQL sources (for MySQL) and I have added the following in `.gitattributes` file:

```
*.sql linguist-language=SQL linguist-detectable=true
```

However, running `github-linguist --breakdown` provides me list of all the SQL sources **except** the ones from the Examples directory although the percentages seem properly calculated as in GitHub. What could be the reason for this?

**Comment by lildude** [comment, marked as answer]
If the files really are in a directory called `Examples` then Linguist will ignore these files too (they won't be included in the stats on GitHub either) as they're considered documentation by default:

https://github.com/github/linguist/blob/299318747a903a90581f46e637f7d3be387ff844/lib/linguist/documentation.yml#L17

You'll also need to implement a documentation override in order for them to be counted.

**Reply by gdsotirov** [reply]
Thanks. I have omitted this information somehow.
